doc/API.md

```diff
@@ -360,9 +503,236 @@ Function: Submit a header that has passed the POW.
 
 Parameters:
 ```
-[]byte
+input []byte
+```
+The input byte array should be 80 bytes that form the solved block header. *Unlike most API calls, it should be written directly to the request body, not as a query parameter.*
```

Comment:
the bytes are random, meaning you can't put the data in as a query string unless you encode them. Encoding them causes blowup, and the miner is typically waiting for the call to return. Even locally, calls like this can take milliseconds, which is a nontrivial amount of time to block the miner.

Basically, hashrates drop when we use querystrings for this call.